serverless/README.md

```diff
@@ -1,190 +1,13 @@
 Serverless Log
 ===============
 
-This is some experimental tooling which allows for the maintenance and querying
-of a log represented by tiles & files. These tools are built upon components
-from the Trillian repo (e.g. compact ranges), but *do not* use the Trillian Log
-service itself.
+Most of the contents of this directory has graduated to its own repo at
+<https://github.com/transparency-dev/serverless-log>.
 
-The idea is to make logging infrastructure a bit more *nix like, and demonstrate
-how to use those tools in conjunction with GitHub actions, GCP Cloud Functions,
-AWS Lambda, etc. to deploy and maintain "serverless" transparency logs.
+The remaining parts are:
+[clone2serverless](./cmd/clone2serverlwss): A tool for dumping transparency
+logs cloned with the [clone](/clone) tools into ai locally hosted serverless
```

Comment:
```suggestion
logs cloned with the [clone](/clone) tools into a locally hosted serverless
```

You're the last person I expected to find putting AI in places it doesn't belong.